### ISSUE TITLE
Server (Phoenix Channels) sending binary data to clients (Slipstream)

### ISSUE THREAD

**Issue by CJRChang** [NFIBrokerage/slipstream#60]
I'm trying to send some binary data from the server (Phoenix Channels) to the client, running Slipstream.

On the server side, this is something like:
```
AppWeb.Endpoint.broadcast!("channel", "topic", {:binary, binary_data})
```
which should invoke `handle_message` on SlipStream side.

Instead this seems to be invoking `handle_info/2` with the following parameters:
```
#1 nil
#2 state
```

It looks like the client side stack trace is something like:
```
deps/slipstream/lib/slipstream.ex:83: ClientApp.Websocket.handle_info/2
lib/slipstream/callback.ex:34
```

**Comment by CJRChang** [comment]
On further investigation, discovered this seems to work for `push/3` and not `broadcast/3`.

**Comment by the-mikedavis** [comment]
Could you give https://github.com/NFIBrokerage/slipstream/pull/61 a try? I believe all we're missing is a function clause to deserialize binary broadcast messages.

**Comment by CJRChang** [comment]
That seems to work now - thanks!

**Comment by the-mikedavis** [comment]
Thanks for checking! This is released now as v1.1.1
